wifiphisher/common/phishinghttp.py

```diff
@@ -19,13 +20,24 @@ def get(self, url):
         self.redirect("http://10.0.0.1:8080/")
 
 
-class ShowPhishingPageHandler(tornado.web.RequestHandler):
+class CaptivePortalHandler(tornado.web.RequestHandler):
 
     def get(self, url):
+        client_request = self.request.path[1:]
         try:
-            if "/" in url:
-                self.redirect("/index")
-            self.render("index.html", **template.get_context())
+            if self.request.path == "/":
```

Comment:
@laozi999 After some testing the only time `self.request.path == "/"` is when you visit `http://10.0.0.1:8080/`. Therefore if we check to see if the file exists this still should work:
```
if user's requested page exists:
    display it to the user
otherwise:
    if any index file is found:
        display it to the user
    otherwise:
        throw an error 
```
Also I noticed you used `os.path.exists` which also returns `True` for both files and directories. You should use `os.path.isfile` instead.